packages/widgets/src/index.js

```diff
@@ -18,11 +18,15 @@ export * from './utils';
  * Note that for the block to be useful, any scripts required by a widget must
  * be loaded into the page.
  *
+ * @param {Object} settings Block settings.
  * @see https://developer.wordpress.org/block-editor/how-to-guides/widgets/legacy-widget-block/
  */
-export function registerLegacyWidgetBlock() {
-	const { metadata, settings, name } = legacyWidget;
-	registerBlockType( { name, ...metadata }, settings );
+export function registerLegacyWidgetBlock( settings = {} ) {
+	const { metadata, settings: defaultSettings, name } = legacyWidget;
+	registerBlockType(
+		{ name, ...metadata },
+		{ ...defaultSettings, ...settings }
+	);
```

Comment:
@noisysocks, what do you think about this method for overriding block registration settings?

Reply:
Looks good to me, but this current implementation will overwrite the block's entire `supports` object when really we only want to overwrite `supports.inserter`. I think we'll want a recursive merge.

```js
import { merge } from 'lodash';
...
registerBlockType(
 		{ name, ...metadata },
 		merge( {}, defaultSettings, settings )
 	);
```

Reply:
Probably adding "lodash" as a dependency to the widgets package for single isn't worth it just. Updated overrides, so it now only changes `supports` settings. I don't think we'll need anything else for now.